fixtures/demo/fixture.mdx

```diff
@@ -0,0 +1,80 @@
+# MDX intellisense
+
+## What is MDX?
```

Comment:
Some of the “marketing” stuff on the MDX site has changed: https://mdxjs.com.
Perhaps some of the text here can match that? Also, perhaps be shorter: no need to show markdown/imports/exports again in detail?

Reply:
I don’t consider myself great at writing demo content. The goal is to provide some content to demo the intellisense features provided by `@mdx-js/monaco`. ESM is definitely part of that, as resolving dependencies can get funky. Markdown link references too, as they are resolved by intellisense. It’s also a nice test case for manual debugging.

Ideally I’d like to host this somewhere as well. (Similar to https://monaco-tailwindcss.js.org / https://monaco-unified.js.org / https://monaco-yaml.js.org)

To see how this is used, run `npm start` from this branch.

Reply:
I am definitely open to suggestions for better content. Also I wouldn’t mind if this gets amended later. I will continue further intellisense development after this PR.